comp/venue/organise-screens.yaml

```
@@ -0,0 +1,39 @@
+summary: Organise display screens for the $SRYYYY competition
+
+priority: should
+
+component: Arena
+
+milestone: $SRYYYY Competition
+
+area-owner: event-logistics
+
+description: >-
+  SR owns 6 matching screens, without stands, which are used for the
+  arena displays. These have the required cables to plug into HDMI sources.
+
+  We also have some 19" screens with stands and a variety of other input types.
+
+  We need to figure out what screens we need for the competition and
+  whether we have enough of the right type.
+
+  For the past few years we have used:
+   - 5 screens at the arena (4 corners & staging).
+   - 2/3 screens for the livestream.
```

I'd encourage using `-` rather than `/` for "two to three" to avoid potential misreading as "two thirds".
```suggestion
   - 2-3 screens for the livestream.
```